quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/SampleApp.java

```diff
@@ -308,7 +308,7 @@ public static void main(String[] args) {
     private static ConfigJson loadConfigs() {
         File configFile = new File(".\\" + SampleApp.configFileName);
         try {
-            ObjectMapper mapper = new ObjectMapper();
+            ObjectMapper mapper = com.microsoft.azure.kusto.data.Utils.getObjectMapper();
```

Comment:
Nitpick , imports + Utils.getObjectMapper()

Reply:
since com.microsoft.azure.kusto.quickstart.Utils already exists , hence we had to use fully qualified class name.